### DISCUSSION TITLE
multiple reminders for measurable tasks

### DISCUSSION THREAD

**Discussion by pokegamer5547** [iSoron/uhabits#924] [Feature Requests]
Example: i want to develop an habit of drinking water, then having multiple reminders which will notify to drink water and record it would be great.

**Comment by cocowash** [comment]
> 
> 
> Example: i want to develop an habit of drinking water, then having multiple reminders which will notify to drink water and record it would be great.

It would be great if for example I could set the interval of those reminders. For example, that the app reminds me to take water from 10am to 11pm each 3 hours.

**Reply by pokegamer5547** [reply]
Yep that's a great idea!

**Reply by Silviopina** [reply]
Ditto! ☺️
